vnext/Chakra/sources.inc

```diff
@@ -17,9 +17,11 @@ SOURCES_SHARED = \
 	..\Utf8DebugExtensions.cpp \
 
 SOURCES_WIN32 = $(SOURCES_SHARED) \
-	..\ChakraRuntime.cpp \
+	..\ChakraJsiRuntime.cpp \
+	..\ChakraJsiRuntime_core.cpp
 
-SOURCES_WINRT = $(SOURCES_SHARED) \
+SOURCES_WINRT = $(SOURCES_SHARED)
 
 SOURCES_JSICORE = \
-	..\ChakraRuntime.cpp \
+	..\ChakraJsiRuntime.cpp \
+	..\ChakraJsiRuntime_core.cpp
```

Comment:
>ChakraJsiRuntime_core [](start = 4, length = 21)

Do we need ChakraJsiRuntime_edgemode.cpp somewhere in this file?  seems to not be built in obuild anywhere.

Reply:
I could get get devmain to build.. but verified only win32 apps .. It was not clear to which devmain target should the edgemode source be included .. I'll work on it once this code reach devmain.